src/js/utils/specs/date-utils-spec.js

```diff
@@ -0,0 +1,145 @@
+                });
+                it('offsets time according to a determined timezone', function() {
+                    var testDate = DateUtils.stringToMoment('2016-10-14 08:00:00+00:00');
+                    // in order to avoid DST messes, we'll just pick some TZs without
+                    var TestZones = {
+                        'America/Winnipeg': 'Oct 14, 2016 3:00 AM',
+                        'Pacific/Honolulu': 'Oct 13, 2016 10:00 PM',
+                        'Asia/Ho_Chi_Minh': 'Oct 14, 2016 3:00 PM',
+                        'Atlantic/Reykjavik': 'Oct 14, 2016 8:00 AM'
+                    };
+                    Object.keys(TestZones).forEach(function(key) {
+                        expect(
+                            new DateUtils.localizeTime(testDate, key).format('lll')
+                        ).toEqual(TestZones[key]);
+                    });
+                });
+            });
+
+            describe('localize', function() {
+                it('tests error conditions', function() {
+                    var context = {};
+                    expect(
+                        DateUtils.localize(context)
+                    ).toEqual('');
+                });
+                it('tests defaults', function() {
+                    var context = {
+                        datetime: '2016-10-14 08:00:00+00:00',
+                        timezone: 'RANDOM_STRING',
+                        language: 'RANDOM_STRING'
+                    };
+                    expect(DateUtils.localize(context)).toEqual('Oct 14, 2016 08:00 UTC');
+                    context = {
+                        datetime: 'RANDOM_STRING',
+                        timezone: 'RANDOM_STRING',
+                        language: 'RANDOM_STRING'
+                    };
+                    expect(DateUtils.localize(context)).toEqual('');
+                });
+
+                it('generates a localized string based on a language preference', function() {
+                    var TestLangs = {
+                        en: 'Oct 14, 2016 08:00 UTC',
+                        ru: '14 окт. 2016 г. 08:00 UTC',
+                        ar: '١٤ تشرين الأول أكتوبر ٢٠١٦ ٠٨:٠٠ UTC'
+                    };
+                    Object.keys(TestLangs).forEach(function(key) {
+                        var context = {
+                            datetime: '2016-10-14 08:00:00+00:00',
+                            timezone: 'UTC',
+                            language: key
+                        };
+                        expect(DateUtils.localize(context)).toEqual(TestLangs[key]);
+                    });
+                });
+                it('checks for formatting', function() {
+                    var TestFormats = {
+                        DEFAULT_TIME_FORMAT: '8:00:00 AM UTC',
+                        DEFAULT_SHORT_DATE_FORMAT: 'Oct 14, 2016',
+                        DEFAULT_LONG_DATE_FORMAT: 'Friday, October 14, 2016 8:00 AM'
+                    };
```

Comment:
Hmm.. why does the `DEFAULT_LONG_DATE_FORMAT` include the time?  The [example in your code](https://github.com/edx/edx-ui-toolkit/pull/85/files#diff-061b8b80c6ba6b948b73a2611b2897e7R51) doesn't.